framework/source/class/qx/ui/virtual/core/Scroller.js

```diff
@@ -86,7 +86,7 @@ qx.Class.define("qx.ui.virtual.core.Scroller",
     // overridden
     _createChildControlImpl : function(id, hash)
     {
-      if (id == "pane") {
+      if (id === "pane") {
```

Comment:
I only changed it because my IDE (PhpStorm) made the background color of it orange and because I know this can be changed without consequences. Line 320 on the other hand, I'm not sure to change it or not.

If you're interested, here's the message my IDE gives:

```
Comparison path != "" may cause unexpected type coercion
This inspection reports usages of JavaScript equality operators which may cause unexpected type coercions. It is considered a good practice to use the type-safe equality operators === and !== instead of their regular counterparts == and !=.
Depending on the option selected, this will either highlight:
All usages of == and != operators.
All usages except comparison with null. Some code styles allow using x == null as a replacement for x === null || x === undefined.
Only suspicious expressions, such as: == or != comparisons to 0, '', null, true, false, or undefined.
```

Reply:
I would prefer === because of type safe comparision.

Reply:
Yes, if you're comparing against the empty string, `""`, it makes a big difference because the empty string is falsy. The type-safe `===` is better in that case. For non-empty strings, it doesn't matter. Regardless, don't worry about this one. Your change is fine.